### PR TITLE
Update Commands documentation

### PR DESCRIPTION
**Pull request by jwflory**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [X] Documentation
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**: Updates Commands documentation, adds tables, improves readability score

# Problem

Gradual modernization of documentation

# Solution

* Follows RST document style guide
* Uses active voice / minor edits
* Reorganize structure, use tables

# Action

Review for content, and if it looks good, merge! 🎉  Once merged, it will go live [here](https://mobarena.readthedocs.io/en/latest/user/commands.html).

### CODE REVIEW

File: docs/user/commands.rst [hunk 1]
```diff
@@ -2,90 +2,117 @@
...
++===========================+============================+=====================+
+| ``/ma join (<arena>)``    | Join arena with given name | ``/ma j (<arena>)`` |
++---------------------------+----------------------------+---------------------+
+| ``/ma leave``             | Leave current arena or     | ``/ma l``           |
+|                           | spectator area             |                     |
++---------------------------+----------------------------+---------------------+
+| ``/ma notready``          | List players who are not   |                     |
+|                           | ready in arena lobby       |                     |
++---------------------------+----------------------------+---------------------+
+| ``/ma spec (<arena>)``    | Enter an arena's spectator | ``/ma s (<arena>)`` |
```
Comment (garbagemule) [inline docs/user/commands.rst]: spectator area* perhaps? :)

**Comment by jwflory** [comment]
@garbagemule Sorry for leaving this hanging for so long. I pushed a new commit to address your feedback. This should be ready to merge. :clapper:

**Comment by jwflory** [comment]
@garbagemule Ping, did you get a chance to look at this one?

**Comment by garbagemule** [comment]
Thanks! :)

**Pull request merged**